src/AmalgamatorClient.ts

```diff
@@ -71,4 +71,9 @@ export class AmalgamatorClient extends DebugClient {
             });
         }
     }
+    public disassembleRequest(
```

Comment:
We'll merge this here, but please create an upstream PR to https://github.com/microsoft/vscode-debugadapter-node/blob/main/testSupport/src/debugClient.ts

If you notice any other missing requests, please add them to that PR too.

Reply:
I have created an upstream PR to https://github.com/microsoft/vscode-debugadapter-node/blob/main/testSupport/src/debugClient.ts. It has merged.

Reply:
Wonderful. Thanks for making https://github.com/microsoft/vscode-debugadapter-node/pull/273